react-accordion-component/my-app/src/Accordion.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+
+export default function Accordion({ topics }) {
+  const [isToggledOn, setToggle] = useState();
```

Comment:
This all looks good. My only complaint is the name is this state variable. `isToggledOn` sounds like it's a boolean because it starts with `is`, but the value is really a topic, not a boolean.